RequestService/RequestService.Handlers/GetRequestsByFilterHandler.cs

```diff
@@ -49,6 +97,37 @@ public async Task<GetRequestsByFilterResponse> Handle(GetRequestsByFilterRequest
 
             var requestSummaries = _repository.GetRequestsByFilter(request, referringGroups);
 
+            if (requestSummaries.Count>0)
+            {
+                await AddPostCodeForLocations(requestSummaries);
+
+                await _jobFilteringService.FilterAllRequests(
```

Comment:
Technically we have the same issue with FilterAllRequests as we had for ExcludeSiblingsOfJobsAllocatedToUserID, i.e. we should be filtering and then expanding back out to whole requests.

_However,_ whilst all Requests have a single location and SupportActivity, the end result will be fine.

Maybe we could just add a comment that this needs to be revisisted in the event that we allow multiple SupportActivities on a single Request?